### PR TITLE
Add some clarifying comments that seemed appropriate while learning the codebase

### PR DESCRIPTION
**Pull request by fitzoh**

### CODE REVIEW

File: expr/tagquery/query.go [hunk 1]
```diff
@@ -20,6 +20,9 @@ func init() {
 	QueryHash = util.NewFnv32aStringWriter
 }
 
+//A Query is a set of graphite tag expressions as used by the graphite `seriesByTag` function
+//tag expression definitions: https://graphite.readthedocs.io/en/latest/tags.html#querying
+//seriesBytTag documentation: https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.seriesByTag
```
Comment (replay) [inline expr/tagquery/query.go]:
```suggestion
//seriesByTag documentation: https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.seriesByTag
```

File: expr/tagquery/query.go [hunk 2]
```diff
@@ -33,6 +36,10 @@ type Query struct {
 	tagClause int
 }
 
+//Parse the list of graphite tag expressions as used by the seriesByTag function
+//tag expression definitions: https://graphite.readthedocs.io/en/latest/tags.html#querying
+//seriesBytTag documentation: https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.seriesByTag
```
Comment (replay) [inline expr/tagquery/query.go]:
```suggestion
//seriesByTag documentation: https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.seriesByTag
```

File: expr/tagquery/tag.go [hunk 3]
```diff
@@ -13,6 +13,8 @@ import (
 //go:generate msgp
 type Tags []Tag
 
+//Parse a list of key value pairs (not including the metric name) into a list of Tags
```
Comment (replay) [inline expr/tagquery/tag.go]: `ParseTags()` may also be used in places where the metric name is specified as a tag, like `name=some.metric.name`

Comment (fitzoh) [reply expr/tagquery/tag.go]: Updated in ea8dbb9af8c9bbf4eeedf564c71df0275f6ae51c, parenthetical isn't necessary with the examples.

**Comment by replay** [comment]
Looks good. Re formatting we generally stick to this comment formatting rule :
```
Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.
```
From: https://golang.org/doc/effective_go.html#commentary

Especially the first word of the comment should be the name being declared.

**Comment by fitzoh** [comment]
>Especially the first word of the comment should be the name being declared.

fd9960caa42ef20405e4bf9a6e223bdb6176b4e5

**Pull request merged**